### ISSUE TITLE
Question about the way bluesky embed thumbnails from an URL

### ISSUE THREAD

**Issue by onmyouji** [bluesky-social/social-app#4241]
**Is your feature request related to a problem? Please describe.**

This is more of a question than a request, but I noticed when user adds a link on their post (e.g. from youtube) and click `Post`, the video thumbnail gets uploaded from user client side (there's upload traffic from my computer).

This is different from other social media like twitter or mastodon. On other social media, when I embed thumbnails from video or other websites, the posts get submitted without requiring any upload from client (users) side.

Is this just how bluesky protocol works?

**Describe the solution you'd like**

The reason I'm asking is because I often get an error when creating a post that has thumbnail. The error message is "Upload timed out, please try again". My location is in Southeast Asia, and the internet peering is kinda bad over here, maybe that's why I got the error.

**Describe alternatives you've considered**

Not sure if it's possible or if there's any technical limitation, but it would be great if users can embed thumbnail on their post without having to upload anything, just like on other websites.

Windows 10 Home 64-bit
Firefox 126.0.1 (64-bit)

**Comment by mozzius** [comment]
Yeah, the way it works is it's on the poster to provide metadata about the link.

It could work differently (and I'd argue maybe it should), but it has the nice upside of being really simple to read the link card if you just have the raw record available, you don't need to go resolve the metadata yourself. So at the moment, we don't have any plans on changing this.

I'm closing this, hope that answers your question
